src/FormElement/ScheduleElement.php

```diff
@@ -565,6 +565,17 @@ private function assembleCommonElements(): void
      */
     public function prepareMultipartUpdate(RequestInterface $request): array
     {
+        $hasErrors = $this->weeklyField->getPopulatedValue('has_errors') === 'y';
```

Comment:
Why do we have transfer `has_errors`? Is there really no other way? Looks fragile.

Reply:
In the normal case we only update the content partially i.e. just the `Recurrence` widget. However, if an error was rendered before for some reason, this message will be displayed forever even if the user has already chosen a valid option in between.

I would be willing to accept better suggestions on how to make this work in a much nicer way.